weekly-updates/Kirstie-Whitaker/Weekly-meetings_2022.md

```diff
@@ -0,0 +1,91 @@
+# Weekly Meetings
+
+Notes from Kirstie's personal version of the "weekly meetings" file she asks all lab members to complete.
+
+* Archives for 2021 can be found in [Weekly-meetings_2021.md](weekly-meetings_2021.md)
+* Archives for 2020 can be found in [Weekly-meetings_2020.md](weekly-meetings_2020.md)
+* Archives for 2019 can be found in [Weekly-meetings_2019.md](weekly-meetings_2019.md)
+* Archives for 2018 can be found in [Weekly-meetings_2018.md](weekly-meetings_2018.md)
+
+**2022**
+
+* [7 January 2022](#date-7-january-2022)
+
+## Date: 7 January 2022
+
+### What brought you joy this week?
+
+* I LOVED attending the DECOVID data paper meeting.
+  I'm so proud of that work!
+  And really helpful to have EP there to give feedback - that was either very minor or positive and supportive!
+  GREAT work from the team 💖
+
+### What did you achieve
+
+* Comments to GT on her fellowship application.
+* AIM community manager roles are now online!!
+  Thank you to SA 🙏
+  * Community Manager, Patient and Public Involvement and Engagement: AIM RSF: https://cezanneondemand.intervieweb.it/turing/jobs/community_manager_patient_and_public_involvement_and_engagement_aim_rsf_19281
+    * Deadline 6 Feb
+  * Community Manager, Open Collaboration: AIM RSF: https://cezanneondemand.intervieweb.it/turing/jobs/community_manager_open_collaboration_aim_rsf_19278
+    * Deadline 6 Feb
+* Kick off meeting with UKHSA.
+* Linked up EDoN folks w CF.
+* A couple of safe haven meeting discussions.
+* Approved training application for AP.
+* Facilitated a conversation in lab meeting about setting goals that got some good feedback.
+* Sent reimbursements to MD.
+* 1:1s with EK, AB, AL, and DC.
+  All a delight.
+* Completed some HR paperwork for AM.
+* Caught up with NG about a couple of topics - rambled a bit but hopefully was helpful!
+* Got few meetings planned & in the calendar.
+
+### What did you struggle with?
+
+* I had intended to get GA comments on her paper and I just didn't manage it at all.
+  NEXT WEEK 💪
+* One project that I'm a co-i on is struggling a bit and I'm not sure of the best way forwards...
+  Needs some reflection and probably some dedicated time investment.
+  (Which is hard to find 😕)
+* Didn't get my butt on linkedin to promote the jobs as I had intended... this could end up biting me if we don't get good folks for the RAM roles 😬
+  * I just really don't like linkedin!!
```

Comment:
I think we should try posting RAM roles here: https://en-gb.facebook.com/groups/1062957250383195/ 

& maybe consider advertising in product manager specific job boards, e.g. https://www.productplan.com/learn/hiring-product-managers/